From 1c96bfa0bc38ce65d52b36723a43d85434913dc9 Mon Sep 17 00:00:00 2001
From: Ricardo Salveti <ricardo@foundries.io>
Date: Thu, 27 Aug 2020 21:44:15 -0300
Subject: [PATCH] [FIO extras] drm: ipuv3: fix missing ret

Add back the ret variable which was removed during the v5.4.70 merge
done by NXP (issue coming from the base BSP tree).

This fixes a build failure for devices enabling drm/imx/ipuv3.

https://github.com/Freescale/linux-fslc/pull/341

Signed-off-by: Ricardo Salveti <ricardo@foundries.io>
---
 drivers/gpu/drm/imx/ipuv3/ipuv3-crtc.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/gpu/drm/imx/ipuv3/ipuv3-crtc.c b/drivers/gpu/drm/imx/ipuv3/ipuv3-crtc.c
index 1f08bff816f84..785280824977f 100644
--- a/drivers/gpu/drm/imx/ipuv3/ipuv3-crtc.c
+++ b/drivers/gpu/drm/imx/ipuv3/ipuv3-crtc.c
@@ -439,6 +439,7 @@ static int ipu_drm_bind(struct device *dev, struct device *master, void *data)
 	struct ipu_client_platformdata *pdata = dev->platform_data;
 	struct drm_device *drm = data;
 	struct ipu_crtc *ipu_crtc;
+	int ret;
 
 	ipu_crtc = dev_get_drvdata(dev);
 	memset(ipu_crtc, 0, sizeof(*ipu_crtc));
-- 
2.31.1

